server_test.go

```diff
@@ -140,6 +142,73 @@ func TestViewConsole(t *testing.T) {
 	}
 }
 
+func TestPowerStatus(t *testing.T) {
+	handler := newHandler()
+	makeNode(t, handler, "somenode", `{
+		"type": "dummy",
+		"info": {
+			"addr": "10.0.0.3",
+			"user": "ipmiuser",
+			"pass": "secret"
+		}
+	}`)
+	token := getToken(t, handler, "somenode")
+	testStringOn := string("{\"power_status\":\"on\"}\n")
+	testStringOff := string("{\"power_status\":\"off\"}\n")
+
+	// First make sure dummy node is off
+	request := requestSpec{"GET",
+		"http://localhost/node/somenode/power_status", ""}
+	resp := tokenReq(handler, token, request)
+	body, _ := ioutil.ReadAll(resp.Body)
```

Comment:
Check the error (here and elsewhere). Always check errors. You can just use errpanic, defined in `util_test.go`.